orchagent/routeorch.cpp

```diff
@@ -1392,11 +1411,6 @@ bool RouteOrch::addRoutePost(const RouteBulkContext& ctx, const NextHopGroupKey
         increaseNextHopRefCount(nextHops);
 
         decreaseNextHopRefCount(it_route->second);
-        if (it_route->second.getSize() > 1
-            && m_syncdNextHopGroups[it_route->second].ref_count == 0)
```

Comment:
Is it easier to insert `bulkNhgReducedRefCnt` here?

Reply:
Keep old if-statement, I think it will keep the `bulkNhgReducedRefCnt` in small size.

---
In reply to: [521673770](https://github.com/Azure/sonic-swss/pull/1501#discussion_r521673770) [](ancestors = 521673770)